src/components/implementation/tests/micro_booter/micro_booter.c

```diff
@@ -15,26 +15,45 @@ term_fn(void *d)
 	SPIN();
 }
 
+static int test_done[NUM_CPU];
+
 void
 cos_init(void)
 {
-	int cycs;
-
-	cos_meminfo_init(&booter_info.mi, BOOT_MEM_KM_BASE, COS_MEM_KERN_PA_SZ, BOOT_CAPTBL_SELF_UNTYPED_PT);
-	cos_compinfo_init(&booter_info, BOOT_CAPTBL_SELF_PT, BOOT_CAPTBL_SELF_CT, BOOT_CAPTBL_SELF_COMP,
-	                  (vaddr_t)cos_get_heap_ptr(), BOOT_CAPTBL_FREE, &booter_info);
+	int cycs, i;
+	static int first_init = 1, init_done = 0;
+	static int cpu_counter = 0;
 
-	termthd = cos_thd_alloc(&booter_info, booter_info.comp_cap, term_fn, NULL);
-	assert(termthd);
+	cos_faa(&cpu_counter, 1);
```

Comment:
I talked to @gparmer about this. We want to keep the kernel and user logic simple around the NUMBER of cores booted up. In order to do that, we just make sure we have as many cores (LAPICs) as we configure in `NUM_CPU`. I've modified that in my smp branch if you want to merge that.